### PR TITLE
[Syntax] Migrate exchange_config_data private method syntax

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: octobot_trading/exchanges/data/exchange_config_data.py [hunk 1]
```diff
@@ -110,15 +113,24 @@ def __set_config_time_frame(self):
...
         return [
             symbol
             for symbol in self.config[CONFIG_CRYPTO_CURRENCIES][crypto_currency][CONFIG_CRYPTO_ADD]
             if self.exchange_manager.symbol_exists(symbol)
-               and symbol not in self.traded_cryptocurrencies_pairs[crypto_currency]
+               and symbol not in self.traded_crypto_currencies_pairs[crypto_currency]
         ]
 
-    def __add_tradable_symbols(self, crypto_currency, symbols):
+    def _create_wildcard_symbol_list(self, crypto_currency):
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/data/exchange_config_data.py]: no need for it in pxd ?

**Comment by Herklos** [comment]
Yes it should be in 👍

File: octobot_trading/exchanges/data/exchange_config_data.py (hunk 1 above)
Comment (Herklos) [reply octobot_trading/exchanges/data/exchange_config_data.py]: Yes it should be in +1

File: tests/exchanges/data/test_exchange_config_data.py [hunk 2]
```diff
@@ -71,33 +71,32 @@ class TestExchangeConfig:
...
+        _, exchange_config = await self.init_default(config=config)
+
+        assert "ICX/BTC" in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
+        assert "NEO/BTC" in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
+        assert "VEN/BTC" in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
+        assert "XLM/BTC" in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
+        assert "ONT/BTC" in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
+        assert "BTC/USDT" in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
+        assert "ETH/USDT" not in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
+        assert "NEO/BNB" not in exchange_config.traded_crypto_currencies_pairs["Bitcoin"]
```
Comment (GuillaumeDSM) [inline tests/exchanges/data/test_exchange_config_data.py]: :+1:

**Pull request merged**